operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 2.0.14
+version: 3.0.0-beta.8
```

Comment:
So I remember that the decision was to make the beta version be just the version from the tip of main. To me that would imply that there's a stable branch here in the mimir repo that contains the chart at the non-beta version. But I don't find this branch here. What am I missing?

Reply:
right, I remembered only partially. This makes sense then. 

Does this change to v3 also commit us to doing all the breaking changes we have lined up until the next mimir release? (Because I assume then we will cut the release and the chart version in this file will be v3)

Reply:
After IRL conversation: use next minor release of mimir as starting point to not fall behind mimir - let's keep helm chart ahead at least - it will deviate anyway due to breaking chart changes.